integration/generator.go

```diff
@@ -0,0 +1,108 @@
+package integration
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/CityOfZion/neo-go/pkg/core/transaction"
+	"github.com/CityOfZion/neo-go/pkg/crypto"
+	"github.com/CityOfZion/neo-go/pkg/crypto/keys"
+	"github.com/CityOfZion/neo-go/pkg/rpc"
+	"github.com/stretchr/testify/require"
+)
+
+// Generator goal is to generate tx for testing purposes.
+type Generator struct {}
+
+func getTransactionsForIntegration(t *testing.T, numberOfTX int) []*transaction.Transaction{
+	var data []*transaction.Transaction
+
+	wif, err := getWif()
+	require.NoError(t, err)
+
+	for n := 0; n < numberOfTX; n++ {
+		//todo: use random TX (like transfers/miner/invocation,...)
+		tx := getTX(t, wif)
+		require.NoError(t, rpc.SignTx(tx, wif))
+		data = append(data, tx)
+	}
+	return data
+}
+
+// getTransactionsForBenchmark returns Invocation txes.
+func getTransactionsForBenchmark(t *testing.B) []*transaction.Transaction {
```

Comment:
I'd probably try to unify it with `getTransactionsForIntegration`

Reply:
that could be done yes, but it's not that obvious. since *testing.B and *testing.T are different and common interface *testing.TB doesn't contain specific functions used by require.

Reply:
> *testing.TB doesn't contain specific functions used by require.

Weird.

Reply:
done